src/render_engine/vulkan-hpp/README.md

```diff
@@ -0,0 +1,4 @@
+# Vulkan HPP Backend
```

Comment:
But why? Why does Nova need two Vulkan backends, one with the C header and one with the C++ header?